wcfsetup/install/files/lib/form/RegisterForm.class.php

```diff
@@ -403,6 +404,14 @@ public function save() {
 			// create fake password
 			$this->password = StringUtil::getRandomID();
 		}
+
+		$data = [
```

Comment:
`$data` is awfully generic and can accidentally clash with future changes to the method.

```suggestion
		$eventParameters = [
```